src/nameserver/name_server_impl.cc

```diff
@@ -8584,6 +8582,22 @@ bool NameServerImpl::AddIndexToTableInfo(const std::string& name, const std::str
         }
     }
     UpdateZkTableNode(table_info);
+    // refresh tablet here, cuz this func may be called by task
```

Comment:
get all tablet by `GetAllHealthTablet` and call refresh

Reply:
tid is just one, I'll fix it, and filter tablet by health and exists in table info